source/_references/operations.md

```diff
@@ -77,7 +74,12 @@ Crops an image to a set size.
     - `XOFFSET` is either a number of pixels or "left", "center", "right"
     - `YOFFSET` is either a number of pixels or "top", "center", "bottom".
    
-   The default value is `auto`, which will crop the image centering the crop box around the defined 
+   Or it can also be
+    - `smart` for smart cropping (where the most attention of the image would be)
+    - `subjectarea` for cropping on the subjectarea, if one exists
+    - `auto`. Default value.
```

Comment:
what's the difference between `auto` and `subjectarea`?

Reply:
ah, the undocumented face detection makes the difference. I added it now to make it clearer (hopefully)